keg_elements/forms/validators.py

```diff
@@ -4,6 +4,34 @@
 
 import jinja2
 from wtforms import ValidationError
+import re
+
+
+class ValidateAlphaNumeric(object):
+    """
+        A validator to make sure than a form field contains only alphanumeric data
+
+        usage example:
+            import keg_elements.forms.validators as validators
+
+            wtforms.StringField('AlphaNumeric', validators=[
+                            validators.ValidateAlphaNumeric()
+                        ])
+    """
+    regex = re.compile(r'^[a-zA-Z0-9][a-zA-Z0-9\s]*$')
```

Comment:
Do we need to use a regex here? Given the tests below, it seems like we could accomplish the same thing just by checking `field.data.isalnum()`.

One piece not handled by `isalnum()` are non-leading spaces, which are permitted by the regex (so `' abc'` fails, but `'abc '` is valid). If these non-leading spaces are necessary, they should probably be added to the tests.

Reply:
`isalnum` will match unicode chars as well. I think this validator is meant to restrict to ascii. If not then the expression will need to be re-written anyway.

Reply:
@tjlevel12 good catch on the white space issue, in this case I don't think we want to support them. @bladams what do you think?